azurerm/internal/services/containers/kubernetes_addons.go

```diff
@@ -248,6 +266,17 @@ func expandKubernetesAddOnProfiles(input []interface{}, env azure.Environment) (
 		}
 	}
 
+	openServiceMesh := profile[""].([]interface{})
```

Comment:
presumably this wants to be:

```suggestion
	openServiceMesh := profile["open_service_mesh"].([]interface{})
```